packages/teleterm/package.json

```diff
@@ -57,19 +57,22 @@
   },
   "productName": "Teleport Connect",
   "build": {
-    "appId": "gravitational.teleport.connect",
+    "appId": "com.gravitational.teleport.connect",
     "asar": true,
     "asarUnpack": "**\\*.{node,dll}",
     "afterSign": "notarize.js",
     "files": [
       "build/app/dist"
     ],
     "mac": {
+      "appId": "QH8AA5B8UP.com.gravitational.teleport.connect",
       "target": "dmg",
       "type": "distribution",
       "hardenedRuntime": true,
       "entitlements": "assets/entitlements.mac.plist",
-      "entitlementsInherit": "assets/entitlements.mac.plist",
+      "entitlementsInherit": "assets/entitlements-inherit.mac.plist",
```

Comment:
Have you tried it without letting the other processes inherit the entitlements? Ideally we don't the permissions to propagate to binaries other than `tsh`.

Reply:
No, I haven't but that's a good point. Matter of fact, we need only the tsh process to have access to those keychain-access-groups as for now it'd be the only process interacting with Touch ID.

I believe the Electron child processes need to have `com.apple.security.cs.allow-jit` set to true and guessing from the errors that I've seen, all child processes must have at least `com.apple.developer.team-identifier` set to the same value as the main process.

I'll see what I can do here.